sinfonier-backend-api/apiRequests/RequestHandler.py

```diff
@@ -3,13 +3,17 @@
 
 class Error(dict):
     def __init__(self, msg):
-        super(Error).__init__()
+        #super(Error).__init__()
```

Comment:
This is something I wanted to ask to you. Why it is needed to? It gives me a wrong arguments exception.